packages/internal-test-helpers/lib/test-resolver.js

```diff
@@ -0,0 +1,43 @@
+import { compile } from 'ember-template-compiler';
+
+class Resolver {
+  constructor() {
+    this._registered = {};
+    this.constructor.lastInstance = this;
```

Comment:
this seems _slightly_ odd, why do we need this (as opposed to keeping track of the instance when we create it)?